app/mailers/report_mailer.rb

```diff
@@ -14,6 +14,32 @@ def budget_status
     )
   end
 
+  def annual_ncr_report(year, to_email)
+    proposals = Proposal.approved.where(client_data_type: "Ncr::WorkOrder")
+    proposals_for_fiscal_year = proposals.select {|p| p.client_data.fiscal_year == year}
+    csv_string = CSV.generate do |csv|
+      csv << ["Amount", "Date Approved", "Org Code", "CL#", 
+        "Budget Activity", "SOC", "Function Code", "Building #", 
+        "Vendor", "Description", "Requestor", "Approver"]
+      proposals_for_fiscal_year.each do |p|
+        approver_name = p.client_data.approving_official ? p.client_data.approving_official.full_name : "no approver listed"
+        csv << [p.client_data.amount, p.root_approval.approved_at, p.client_data.org_code, 
+          p.client_data.cl_number, p.client_data.expense_type, p.client_data.soc_code, 
+          p.client_data.function_code, p.client_data.building_number, p.client_data.vendor, 
+          p.client_data.description, p.requester.full_name, approver_name]
+      end
+    end
+
+    attachments['NCR_Work_Order_Report_FY' + year.to_s + '.csv'] = csv_string
```

Comment:
Minor: the convention in Ruby is to use string interpolation rather than concatenation. In other words,

```ruby
"NCR_Work_Order_Report_FY#{year.to_s}.csv"
```